scripts/setup_devmode.sh

```diff
@@ -44,9 +44,10 @@ CFG_TMPDIR = u'/tmp'
 COLLECT_STORAGE = u'flask_collect.storage.link'
 EOF
 
+echo "Building Assets, bower install"
```

Comment:
Maybe just call the message `inveniomanage bower/collect/assets` for the sake of harmony with earlier `inveniomanage database create` messages.